docs/UserGuide.md

```diff
@@ -2,16 +2,29 @@
 layout: page
 title: User Guide
 ---
+# TutorBuddy
+Welcome to the user guide for TutorBuddy!<br/>
+Are you a tuition teacher looking for a reliable way to keep track of
+your 1-to-1 lessons? Needing a system to remind you about upcoming classes?
+Or an efficient way to calculate your class incomes? <br/>
```

Comment:
Hi shion! I think you have updated the class at some other places. I think updating it here under class income will be good as well. (E.g. Lesson incomes instead class incomes)

Reply:
omg yes sry thanks for pointing out